providers/amazon/src/airflow/providers/amazon/aws/hooks/sagemaker_unified_studio.py

```diff
@@ -0,0 +1,188 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""This module contains the Amazon SageMaker Unified Studio Notebook hook."""
+
+import time
+
+from sagemaker_studio import ClientConfig
+from sagemaker_studio._openapi.models import GetExecutionRequest, StartExecutionRequest
```

Comment:
Why are we importing from private path? If upstream library decided `_openapi` is private we should not import it in Airflow.

Reply:
So the OpenAPI spec in the SageMaker Studio SDK is auto-generated by the OpenAPI generator: https://github.com/openapitools/openapi-generator

I don't think the intention is actually to keep it private but I'll ask the package owners why they named it as such.

Reply:
They need to change it otherwise we can't use it.

Reply:
Confirmed with the team it'll be changed. Made revisions here to reflect those changes.

Reply:
@eladkal Changes are live, no more private package imports